assets/app/scripts/directives/catalog.js

```diff
@@ -1,66 +1,49 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .directive('catalogTemplate', function($location) {
+  .directive('catalogTemplate', function($location, createFromTemplateURLFilter) {
     return {
       restrict: 'E',
+      // Replace the catalog-template element so that the tiles are all equal height as flexbox items.
+      // Otherwise, you have to add the CSS tile classes to catalog-template.
+      replace: true,
       scope: {
         template: '=',
         project: '='
       },
       templateUrl: 'views/catalog/_template.html',
       link: function(scope, elem, attrs) {
-        $(".select-template", elem).click(function() {
-          // Must trigger off of the modal's hidden event to guarantee modal has finished closing before switching screens
-          $(".modal", elem).on('hidden.bs.modal', function () {
-            scope.$apply(function() {
-              var createURI = URI.expand("project/{project}/create/fromtemplate{?q*}", {
-                project: scope.project,
-                q: {
-                  name: scope.template.metadata.name,
-                  namespace: scope.template.metadata.namespace
-                }
-              });
-              $location.url(createURI.toString());
-            });
-          })
-          .modal('hide');
-
+        elem.click(function() {
```

Comment:
@liggitt I kept the click handler here for now. Can't use `href` on a `div`, and we have content inside the tile that's not allowed inside an anchor element. I could possibly add `ng-click` to the `catalog-template` element. Thoughts?

Same thing for builder images below.